mlflow/genai/scorers/builtin_scorers.py

```diff
@@ -212,71 +254,43 @@ def groundedness():
+        result = mlflow.genai.evaluate(data=data, scorers=[groundedness])
     """
-    return _Groundedness()
-
 
-class _GuidelineAdherence(_BaseBuiltInScorer):
-    name: str = "guideline_adherence"
-    global_guidelines: Optional[list[str]] = None
-    required_columns: set[str] = {"inputs", "outputs"}
-
-    def update_evaluation_config(self, evaluation_config) -> dict:
-        # Metric name should always be "guideline_adherence" regardless of the custom name
-        config = deepcopy(evaluation_config)
-        metrics = config.setdefault(GENAI_CONFIG_NAME, {}).setdefault("metrics", [])
-        if "guideline_adherence" not in metrics:
-            metrics.append("guideline_adherence")
+    name: str = "groundedness"
+    required_columns: set[str] = {"inputs", "outputs", "retrieved_context"}
 
-        # If global guidelines are specified, add it to the config
-        if self.global_guidelines:
-            # NB: The agent eval harness will take multiple global guidelines in a dictionary format
-            #   where the key is the name of the global guideline judge. Therefore, when multiple
-            #   judges are specified, we merge them into a single dictionary.
-            #   https://docs.databricks.com/aws/en/generative-ai/agent-evaluation/llm-judge-reference#examples
-            global_guidelines = config[GENAI_CONFIG_NAME].get("global_guidelines", {})
-            global_guidelines[self.name] = self.global_guidelines
-            config[GENAI_CONFIG_NAME]["global_guidelines"] = global_guidelines
+    def __call__(
+        self, *, inputs: Any, outputs: Any, retrieved_context: list[dict[str, Any]]
+    ) -> Assessment:
+        """Evaluate groundedness of response against context."""
+        return super().__call__(inputs=inputs, outputs=outputs, retrieved_context=retrieved_context)
 
-        return config
+    def with_config(self, *, name: str = "groundedness") -> "Groundedness":
+        """
+        Get a new scorer instance with a specified name.
 
-    def validate_columns(self, columns: set[str]) -> None:
-        super().validate_columns(columns)
-        # If no global guidelines are specified, the guidelines must exist in the input dataset
-        if not self.global_guidelines and "expectations/guidelines" not in columns:
-            raise MissingColumnsException(self.name, ["expectations/guidelines"])
+        Args:
+            name: The name of the scorer. Default is "groundedness".
 
-    def __call__(
-        self,
-        *,
-        inputs: Any,
-        outputs: Any,
-        guidelines: dict[str, list[str]],
-        guidelines_context: dict[str, Any],
-    ) -> Assessment:
-        """Evaluate adherence to specified guidelines."""
-        return super().__call__(
-            inputs=inputs,
-            outputs=outputs,
-            guidelines=guidelines,
-            guidelines_context=guidelines_context,
-        )
+        Returns:
+            The updated Groundedness scorer instance.
```

Comment:
`updated` sounds like the caller gets updated.

```suggestion
            A new Groundedness scorer instance with the spcified name.
```

Reply:
makes sense!